pkg/queue/concurrency_state.go

```diff
@@ -81,9 +92,9 @@ func ConcurrencyStateHandler(logger *zap.SugaredLogger, h http.Handler, pause, r
 		}
 
 		logger.Info("Requests increased from zero")
-		if err := resume(); err != nil {
+		if errCode, err := resume(); errCode != noError {
```

Comment:
same comment, this would be better done using the existing `err` return (i.e. if `err == nil { .. }`), no need to add or use a new return parameter here.

Reply:
oh, ok, got it